lib/tasks/top_records.rake

```
@@ -0,0 +1,166 @@
+    puts
+    puts "Value Tier Breakdown:"
+    puts "-" * 50
+    tiers = [
+      ["$1,000+", 1000],
+      ["$500-$999", 500],
+      ["$250-$499", 250],
+      ["$100-$249", 100],
+      ["$50-$99", 50],
+      ["$25-$49", 25],
+      ["$10-$24", 10],
+      ["Under $10", 0]
```

RuboCop (1.84.2)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)